src/domain/data-decoder/entities/schemas/data-decoded.schema.ts

```diff
@@ -0,0 +1,28 @@
+import { JSONSchemaType, Schema } from 'ajv';
+import { DataDecodedParameter } from '../data-decoded.entity';
+
+export const dataDecodedParameterSchema: JSONSchemaType<DataDecodedParameter> =
```

Comment:
Any feedback on this schema would be welcomed. I tried to be flexible when defining this because I think the data decoded payload has a pretty open format. But maybe I lack of the knowledge to define a more fine-grained schema.

Reply:
Of course if you feel we need to change any definition here, I'll include the changes in the entity and OpenAPI definitions as well 🙂